tsconfig.json

```diff
@@ -12,8 +12,7 @@
 		"noPropertyAccessFromIndexSignature": true,
 		"noUncheckedIndexedAccess": true,
 		"strict": true,
-		"noEmitOnError": true,
-		"tsBuildInfoFile": "./dist/tsconfig.tsbuildinfo"
```

Comment:
Umm why did these get deleted

Reply:
The `tsBuildInfoFile` setting can't be set if `incremental` or `composite` are set, since it won't generate a `tsBuildInfoFile` without one of those being set.